titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/endpoint/v3/grpc/JobAuthorizationService.java

```diff
@@ -75,10 +68,6 @@ protected JobAuthorizationService(Predicate<String> callerIdPredicate) {
     }
 
     private Mono<AuthorizationStatus> authorizeCaller(JobDescriptor<?> jobDescriptor, String originalCallerId) {
-        if (!callerIdPredicate.test(originalCallerId)) {
```

Comment:
Does it mean, we do not support user-level security domains?

Reply:
We do but the check is made in the implementing class.